packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,127 @@
+| ----------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| **children**      | `React.ReactNode`                                                        | `undefined` | Accepts multiple `Chip` components                                                                                             | ✅       |
+| **defaultValue**  | `string \|string[]`                                                      | `undefined` | Sets the initial value of the Chip group                                                                                       | ❌       |
+| **isDisabled**    | `boolean`                                                                | `false`     | Disables or enables ChipGroup, it will propagate down to all the Chips                                                         | ❌       |
+| **name**          | `string`                                                                 | `undefined` | The name of the chip group, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name) | ❌       |
+| **onChange**      | `({ name, value }: { name: string; value: string \| string[] }) => void` | `undefined` | Calls a function on any state change within the ChipGroup                                                                      | ❌       |
+| **selectionType** | `'single' \| 'multiple'`                                                 | `'single'`  | Defines the selection behavior within the ChipGroup component                                                                  | ❌       |
+| **size**          | `'xsmall' \| 'small' \| 'medium' \| 'large'`                             | `'small'`   | Specifies the size of the rendered Chips withing the ChipGroup                                                                 | ❌       |
+| **value**         | `string \|string[]`                                                      | `undefined` | Acts as a controlled component by specifying the ChipGroup value                                                               | ❌       |
+| **variant**       | `'positive' \| 'negative' \| 'notice' \| 'information' \| 'neutral'`     | `'neutral'` | Sets the ChipGroups's visual variant, it will propagate down to all the Chips                                                  | ❌       |
+
+## Examples
+
+Here are a few illustrative examples showcasing the utilization of the `Chip` and `ChipGroup` components:
+
+### Basic/Uncontrolled Usage
+
+Single Chip selection:
+
+```jsx
+<ChipGroup defaultValue="1">
+  <Chip value="1">Single chip</Chip>
+  <Chip value="2">Can be selected</Chip>
+  <Chip value="3">At a time</Chip>
+</ChipGroup>
```

Comment:
please add better example in Razorpay's context

Reply:
Updated.